src/components/Header/style.css

```diff
@@ -0,0 +1,10 @@
+.logo {
+    display: inline-block;
+    width: auto;
+    height: 1em;
+    margin-right: 0.3em;
+  }
+
+h1 {
+    text-align: center;
+}
```

Comment:
nevím jestli si o tom s luďkem ještě budeme povídat (=jestli je to v osnově), ale dej si bacha na to že i když css soubor importujeme v komponentě (Header/index.js) tak se pravidla v něm vztahují na celou stránku/aplikaci. **Tj tímhle styluješ všechny nadpisy h2 ve všech komponentách v celé aplikaci**. Proto v těch příkladech před a po luděk boužívá bem pojmenování tříd (movie__detail, movie__title, ...).

Reply:
kdybys to chtěla vylepšít, tak 
```
const Header = ({text}) => {
    return (
        <h1 className="header">
            <img className="logo" src={camera} alt="Logo" />
            {text}
        </h1>
    )
};
...
.header .logo {
    display: inline-block;
    width: auto;
    height: 1em;
    margin-right: 0.3em;
  }

h1.header {
    text-align: center;
} 
```